liquid-rust-syntax/src/surface.rs

```diff
@@ -260,6 +302,260 @@ impl Desugar {
     }
 }
 
-pub fn desugar(ssig: FnSig) -> ast::FnSig {
+pub fn desugar(ssig: BareFnSig) -> ast::FnSig {
     Desugar::desugar(ssig)
 }
+
+// ---------------------------------------------------------------------------
```

Comment:
Everything related to default signatures should probably be in its own module. maybe `liquid-rust-syntax::default`